experimental/web/sample_dynamic/index.html

```diff
@@ -248,7 +248,7 @@ <h2>Compile your own program</h2>
       const functionName = functionNameInput.value;
       const functionArguments = functionArgumentsInput.value.split("\n");
 
-      ireeCallFunction(loadedProgram, functionName, functionArguments)
+      iree-compileallFunction(loadedProgram, functionName, functionArguments)
```

Comment:
revert this line (`ireeCallFunction` is the function name, find-replace probably hit that accidentally)

Reply:
Oops. That must have been before I hit the case sensitive button.